src/game/server/gamecontext.h

```diff
@@ -220,6 +220,8 @@ class CGameContext : public IGameServer
 	char m_aVoteCommand[VOTE_CMD_LENGTH];
 	char m_aVoteReason[VOTE_REASON_LENGTH];
 	int m_NumVoteOptions;
+	int m_Ragequit;
```

Comment:
This shite needs to move to pplayer. Theze variables are for the whole game.

Reply:
I also think you need to initialize m_Ragequit to 0 somewhere (player constructor?)

Reply:
yes and yes